src/Graph.elm

```diff
@@ -1263,7 +1223,7 @@ topologicalSort (AcyclicGraph graph ordering) =
         error =
             "Graph.topologicalSort: Invalid `AcyclicGraph`, where the ordering contained nodes not present in the graph"
     in
-    List.map (\id -> unsafeGet error id graph) ordering
+    List.filterMap (\id -> get id graph) ordering
```

Comment:
This will silently filter out ids that don't exist

Reply:
Keep the call to `unsafeGet` and make `unsafeGet` call `crashLoop`